src/main/java/org/openrewrite/java/migrate/JpaCacheProperties.java

```diff
@@ -154,10 +154,26 @@ public Xml visitTag(Xml.Tag tag, ExecutionContext ctx) {
             // if we could determine an appropriate value, create the element.
             if (scmValue != null) {
                 if (!v1) {
-                    Xml.Tag newNode = Xml.Tag.build("<shared-cache-mode>" + scmValue + "</shared-cache-mode>");
+                    //Xml.Tag newNode = Xml.Tag.build("<shared-cache-mode>" + scmValue + "</shared-cache-mode>");
                     // Ideally we would insert <shared-cache-mode> before the <validation-mode> and <properties> nodes
                     Cursor parent = getCursor().getParentOrThrow();
-                    t = autoFormat(addOrUpdateChild(t, newNode, parent), ctx, parent);
+                    List<Xml.Tag> newChildren = new ArrayList<>();
+                    Xml.Tag sharedCacheTag = Xml.Tag.build("<shared-cache-mode>" + scmValue + "</shared-cache-mode>").withPrefix("\n        ");
```

Comment:
Setting the prefix here is not necessary.
```suggestion
                    Xml.Tag sharedCacheTag = Xml.Tag.build("<shared-cache-mode>" + scmValue + "</shared-cache-mode>");
```